CM.Text/BusinessMessaging/MessageBuilder.cs

```diff
@@ -113,5 +113,20 @@ public MessageBuilder WitHybridAppKey(Guid appKey)
             this._message.HybridAppKey = appKey;
             return this;
         }
+
+        /// <summary>
+        ///  Adds a whatsapp template message that replaces the <see cref="Message.Body" /> for whatsApp
```

Comment:
I'd suggest to use PascalCasing for WhatsApp in both cases, and replace the wa abbreviation with WhatsApp written fully